bucket/fq.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "0.0.4",
+    "description": "Tool, language and decoders for working with binary data.",
+    "homepage": "https://github.com/wader/fq",
+    "license": {
+        "url": "https://github.com/wader/fq/blob/master/LICENSE"
```

```suggestion
    "license": {
        "identifier": "Unknown",
        "url": "https://github.com/wader/fq/blob/master/LICENSE"
```

Looks like we need this field too to pass the CI